x-pack/plugin/core/src/main/resources/logs-default-template.json

```diff
@@ -0,0 +1,15 @@
+{
+  "index_patterns": ["logs-*-*"],
+  "priority": 100,
```

Comment:
Should we set priority to 1 or 10?

Reply:
Definitely not 1, because we want ample space for lower-priority templates, 100 seemed reasonable and was confirmed by the core/features team as a reasonable default. I'd prefer to stick with 100